graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLWebAutoConfiguration.java

```diff
@@ -126,6 +127,9 @@
     @Autowired(required = false)
     private MultipartConfigElement multipartConfigElement;
 
+    @Autowired(required = false)
+    private GraphQLBatchExecutionHandlerFactory graphQLBatchExecutionHandlerFactory;
```

Comment:
what happens if GraphQLBatchExecutionHandlerFactory is not injected? can batched query still be processed ?

Reply:
yes, there is a default in the config object and it is used if none/null is passed in. Default is previous behavior